blockware/pixel-dust/src/main.cpp

```diff
@@ -5,7 +5,7 @@
 #include <SPI.h>
 #include <Text.h>
 #include <Adafruit_PixelDust.h>
-#include <SparkFun_MMA8452Q.h>
+#include <LIS2DW12Sensor.h>
```

Comment:
I wonder if we should ifdef this so we can still build for our prototype blocks :)